cmd/worker/main.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/go-logr/logr"
+	"github.com/kubernetes-sigs/kernel-module-management/internal/cmd"
+	"github.com/kubernetes-sigs/kernel-module-management/internal/worker"
+	"github.com/spf13/cobra"
+	"k8s.io/klog/v2"
+	"k8s.io/klog/v2/klogr"
+)
+
+var (
+	configHelper = worker.NewConfigHelper()
+	logger       logr.Logger
+	w            worker.Worker
+)
+
+var rootCmd = &cobra.Command{
+	Use:               "worker",
+	CompletionOptions: cobra.CompletionOptions{DisableDefaultCmd: true},
+}
+
+var kmodCmd = &cobra.Command{
+	Use:   "kmod",
+	Short: "Manage kernel modules",
+}
+
+var kmodLoadCmd = &cobra.Command{
+	Use:   "load",
+	Short: "Load a kernel module",
+	Args:  cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cfgPath := args[0]
+
+		logger.V(1).Info("Reading config", "path", cfgPath)
+
+		cfg, err := configHelper.ReadConfigFile(cfgPath)
```

Comment:
Didn't we say the config file is a pod annotation?

Reply:
We did. That annotation is mounted as a file in the pod's filesystem. This is the code running in the worker pod, it reads that config file.